Create dnslookupcheck.py

jfletchersimpson opened the pull request:
new dnslookup check collector.

Commit:
Create dnslookupcheck.py

new dnslookup check collector

Grumblesmith reviewed (changes requested):
The default_config and default_config_help need corrections.

TheConnMan commented on src/collectors/dnslookupcheck/dnslookupcheck.py:
```diff
@@ -0,0 +1,74 @@
+        """
+
+        #check to see if the dns name returns an IP address
+        for x in self.config['dnsAddressList']:
```
`x` is not a descriptive variable name. I recommend changing to something like `dnsAddress`.

Commit:
DNSLookupCheckCollector loop variable improvement

Setting a better variable name for the collection loop in
DNSLookupCheckCollector

Commit:
Corrections to DNSLookupCheckCollector

Adding proper default_config and default_config_help sections to the
DNSLookupCheckCollector

Commit:
DNSLookupCheckCollector styling

Restricting line lengths to 79 characters, per PEP 8

Grumblesmith commented:
@TheConnMan I've updated the variable names, default_config and default_config_help sections, and shortened several lines which overran the PEP 8 line length limit. Could you take another look?

TheConnMan commented:
I was already looking at it.

Grumblesmith commented:
Thanks!
